coala_quickstart/generation/SettingsClass.py

```diff
@@ -0,0 +1,195 @@
+import inspect
+
+
+def get_default_args(func):
+    """
+    :param func: Function name.
```

Comment:
Can you add some doctests like https://github.com/coala/coala-quickstart/blob/master/coala_quickstart/info_extraction/Utilities.py#L7?